internal/executor/configuration/podchecks/types.go

```diff
@@ -26,8 +26,9 @@ const (
 )
 
 type Checks struct {
-	Events            []EventCheck
-	ContainerStatuses []ContainerStatusCheck
+	Events             []EventCheck
+	ContainerStatuses  []ContainerStatusCheck
+	DeadlineForUpdates time.Duration
```

Comment:
I think i have one more suggestion here (the fault is mine for making the original suggestion, my apologies).  "deadline" is usually a time.Time set at some point in the future, and "timeout" a duration. Is it worth making this a Time, just to match the usual convention?